snappy/snap_remote_repo.go

```diff
@@ -383,3 +386,61 @@ func (s *SnapUbuntuStoreRepository) Updates() (parts []Part, err error) {
 func (s *SnapUbuntuStoreRepository) Installed() (parts []Part, err error) {
 	return nil, err
 }
+
+// Download downloads the given snap and returns the filename of the local
+// version
```

Comment:
```
// Download downloads the given snap and returns its filename.
// The file is saved in temporary storage, and should be removed
// after use to prevent the disk from running out of space.
```